client/lang/src/en.json

```diff
@@ -9,5 +9,14 @@
     "MFAWebAuthnRegister.REGISTERING": "Registering",
     "MFAWebAuthnRegister.RETRY": "Retry",
     "MFAWebAuthnRegister.SUCCESS": "Key verified",
-    "MFAWebAuthnRegister.WAITING": "Waiting"
+    "MFAWebAuthnRegister.WAITING": "Waiting",
+
+    "MFAWebAuthnVerify.DESCRIPTION": "Use your secuirty key to verify your identity",
+    "MFAWebAuthnVerify.FAILURE": "Something went wrong, Please re-insert your key and try again",
+    "MFAWebAuthnVerify.HELP": "How to use security keys.",
+    "MFAWebAuthnVerify.INSTRUCTION": "Please insert your security key and {button}. ",
```

Comment:
```suggestion
    "MFAWebAuthnVerify.INSTRUCTION": "Please insert your security key and {button}.",
```

Reply:
You found my cheeky 'hack' to stop the help text pushing up on the instruction text.
I'll see if there's a missing class or something I can add to stop it in a different way.

Reply:
I think we've used a span with a margin in other parts of the MFA codebase

Reply:
I've got no idea why this doesn't work. As I suspected that's not a valid fix anyway so I removed it.
Could be a rendering bug in Chrome?
So far as I'm aware two inline nodes next to each other should blend their text in a normal fashion, meaning there'd be a space.
but...
![image](https://user-images.githubusercontent.com/778003/60066123-59be3800-975a-11e9-9f30-5a3e65d2e227.png)

Any ideas?

Reply:
I'm pretty sure it's React that is removing the space for you. As I say, we've used a span with a margin to fix this in other parts of MFA

Reply:
I just looked up two examples of this kind of thing - `TOTP/Register` and `MFA/MethodTile`. Both of them have hard coded spaces on the end of the string to get around this. Maybe just continue and do that for now.